iml-system-test-utils/src/lib.rs

```diff
@@ -273,14 +273,22 @@ impl Config {
     pub fn get_setup_config(&self) -> String {
         match &self.test_type {
             TestType::Rpm => format!(
-                r#"USE_STRATAGEM = {}
-BRANDING = "{}""#,
+                r#"USE_STRATAGEM={}
+BRANDING={}
+LOG_LEVEL=10
+RUST_LOG=debug
```

Comment:
Just a note, this makes less noise:

```
RUST_LOG=debug,hyper=warn,lapin=warn,warp=info
```

Reply:
If we find that we need to change this i'm willing to do so in a separate patch.

Reply:
I think we will actually want the verbosity in this case.